Source/Core/Core/AchievementManager.cpp

```diff
@@ -1184,6 +1209,30 @@ AchievementManager::PingRichPresence(const RichPresence& rich_presence)
   return r_type;
 }
 
+void AchievementManager::DisplayWelcomeMessage()
+{
+  std::lock_guard lg{m_lock};
+  PointSpread spread = TallyScore();
+  if (hardcore_mode_enabled)
+  {
+    OSD::AddMessage(fmt::format("You have {}/{} achievements worth {}/{} points",
+                                spread.hard_unlocks, spread.total_count, spread.hard_points,
+                                spread.total_points),
+                    OSD::Duration::VERY_LONG, OSD::Color::YELLOW,
+                    (Config::Get(Config::RA_BADGES_ENABLED)) ? m_game_badge.osd_icon : nullptr);
+    OSD::AddMessage("Hardcore mode is ON", OSD::Duration::VERY_LONG, OSD::Color::YELLOW);
+  }
+  else
+  {
+    OSD::AddMessage(fmt::format("You have {}/{} achievements worth {}/{} points",
+                                spread.hard_unlocks + spread.soft_unlocks, spread.total_count,
+                                spread.hard_points + spread.soft_points, spread.total_points),
+                    OSD::Duration::VERY_LONG, OSD::Color::CYAN,
+                    (Config::Get(Config::RA_BADGES_ENABLED)) ? m_game_badge.osd_icon : nullptr);
+    OSD::AddMessage("Hardcore mode is OFF", OSD::Duration::VERY_LONG, OSD::Color::CYAN);
+  }
+}
+
```

Comment:
You didn't remove the equivalent code from `LoadGameByFilenameAsync()` that I assume you were intending to replace with this.

Reply:
A lot of these were things from previous attempts I forgot to remove but this makes it obvious I borked a rebase somehow, my mistake.